lib/ui/page/home/page/chat/view.dart

```diff
@@ -1429,29 +1379,32 @@ class ChatView extends StatelessWidget {
           children: [
             Row(
               children: [
+                AnimatedSize(
+                  duration: 150.milliseconds,
+                  child: c.selecting.value
+                      ? const SizedBox(key: Key('Expanded'), width: 40)
+                      : const SizedBox(),
+                ),
                 Expanded(
                   child: c.selecting.value
                       ? SelectionContainer.disabled(
                           child: IgnorePointer(child: child),
                         )
                       : child,
                 ),
-                if (!overlay)
-                  AnimatedSize(
-                    duration: 150.milliseconds,
-                    child: c.selecting.value
-                        ? const SizedBox(key: Key('Expanded'), width: 32)
-                        : const SizedBox(),
-                  ),
               ],
             ),
             Positioned.fill(
               child: Align(
-                alignment: Alignment.centerRight,
+                alignment: Alignment.centerLeft,
                 child: c.selecting.value
                     ? Padding(
                         padding: const EdgeInsets.only(left: 8),
-                        child: SelectedDot(selected: selected, darken: 0.1),
+                        child: SelectedDot(
+                          inverted: false,
+                          selected: selected,
+                          darken: 0.1,
+                        ),
```

Comment:
Also why aren't these dots centered? I guess that while you were developing it you saw that and suggested that this way it looks or behaves better. Why?

<img width="597" height="267" alt="Screenshot 2025-09-17 at 12 34 58" src="https://github.com/user-attachments/assets/507fcb0b-dd6e-4e57-bfb6-c6ba2367d1b8" />